src/core/MOM.F90

```diff
@@ -1674,8 +1674,8 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
                  "faster by eliminating subroutine calls.", default=.false.)
   call get_param(param_file, "MOM", "DO_DYNAMICS", CS%do_dynamics, &
                  "If False, skips the dynamics calls that update u & v, as well as "//&
-                 "the gravity wave adjustment to h. This is a fragile feature and "//&
-                 "thus undocumented.", default=.true., do_not_log=.true. )
+                 "the gravity wave adjustment to h. This may be a fragile feature, "//&
+                 "but can be useful during development", default=.true.)
```

Comment:
Note to GFDL: the change to the parameter documentation requires MOM6-examples to be updated after the merge.